### PR TITLE
Spelling Updates Batch 6

### PR DESCRIPTION
**Pull request by kingthorin**
Based on: zaproxy/zap-extensions#3083 

* against
* aggregated
* decimal
* delimiters
* detected
* encoding
* expires
* multiple
* programmatically
* relevant
* reports
* response
* responseHas
* servlet
* should
* state
* threshold
* tighter
* unknown
* unused
* value
* verify
* without

Signed-off-by: Josh Soref <jsoref@users.noreply.github.com>

### CODE REVIEW

**Comment by kingthorin** [comment]
Failure seems unrelated, persisted across multiple runs, will try again later.

**Comment by kingthorin** [comment]
Build passed on the latest attempt.

File: addOns/pscanrulesAlpha/src/main/java/org/zaproxy/zap/extension/pscanrulesAlpha/Base64Disclosure.java [hunk 1]
```diff
@@ -67,7 +67,7 @@ public class Base64Disclosure extends PluginPassiveScanner {
     }
 
     /**
-     * patterns used to identify strings withut each of the given character sets which is used to
+     * patterns used to identify strings without each of the given character sets which is used to
```
Comment (thc202) [inline addOns/pscanrulesAlpha/src/main/java/org/zaproxy/zap/extension/pscanrulesAlpha/Base64Disclosure.java:70]: Changelog should be updated (e.g. `Maintenance changes.`).

**Comment by kingthorin** [comment]
CHANGELOG note added.

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you both!

**Review by psiinon — APPROVED** [review]

**Pull request merged**